src/webui/www/private/views/rss.html

```diff
@@ -503,9 +503,12 @@
                     if ((rssFeedRows.length - 1) === flattenedResp.length) {
                         match = true;
                         for (let i = 0; i < flattenedResp.length; ++i) {
-                            if (((flattenedResp[i].uid ? flattenedResp[i].uid : "") !== rssFeedRows[i + 1].full_data.dataUid)
-                                || (flattenedResp[i].fullName !== rssFeedRows[i + 1].full_data.dataPath)
-                                || (flattenedResp[i].url !== rssFeedRows[i + 1].full_data.dataUrl)) {
+                            const newData = flattenedResp[i];
+                            const existingData = rssFeedRows[i + 1].full_data;
+                            if ((newData.fullName !== existingData.dataPath)
+                                // folders don't have the following properties
```

Comment:
I think you meant: "only non-folders have the following properties" because you were not checking for folders but other below.

Reply:
IMO it is bad to break the if predicates in the middle. You can just omit it or move it to the end. The code is already clear that it is checking for folder.
```suggestion
                                ...code... // folders don't have the following properties
```

Reply:
I reordered this but left the comment in. I think it's useful to note why we're only performing this check on non-folders - specifically because folders don't have these properties at all (as opposed to some other reason).